README.md

```diff
@@ -79,11 +79,11 @@ The default location for the s3fs password file can be created:
 * using a .passwd-s3fs file in the users home directory (i.e. ~/.passwd-s3fs)
 * using the system-wide /etc/passwd-s3fs file
 
-Enter your S3 identity and credential in a file `~/.passwd-s3fs` and set
+Enter your AWS credentials in a file `~/.passwd-s3fs` and set
```

Comment:
This wording is OK although s3fs supports non-AWS S3 implementations.  Perhaps just say credentials here and `ACCESS_KEY_ID` and `SECRET_ACCESS_KEY` below?

Reply:
That makes sense, I'll make the change.